src/App.tsx

```diff
@@ -1,41 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
+  const [idOfUser, setIdOfUser] = useState<number>(0);
+  const [selectedPostId, setSelectedPostId] = useState<number>(0);
+  const [post, setPost] = useState<Post | null>(null);
```

Comment:
You shouldn't store the post in App state
Only selectedPostId

Reply:
you sure about this point ?